apps/backend/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM node:14 as base
```

Comment:
Can't we have a single NodeJS version? It would simplify the Dockerfile

Reply:
Because we need first to prepare the workspace stuff as it is a monorepo.